.github/workflows/jekyll.yml

```diff
@@ -1,8 +1,10 @@
 name: Jekyll build site
 
 on:
-  pull_request:
-    branches: [ master ]
+  push:
+    branches-ignore:
+      - 'master'
+      - 'production'
```

Comment:
So this action will happen on any branch *except for* master and production: https://stackoverflow.com/questions/57699839/github-actions-how-to-target-all-branches-except-master

Not sure why, I was expecting: 

```
on:
  push:
    branches:
      - 'production'
```

I imagine I'm missing something, please enlighten me @nikolai-b !

Reply:
I didn't want the bot to do any stuff on master or production, in case I've made a mistake (or it does).  It felt like a bad idea to allow the bot to push code that would go to production (or even staging) without a human reviewing it.